utils/check_bad_commit.py

```diff
@@ -47,6 +47,9 @@ def create_script(target_test):
 
 if len(result.stderr) > 0:
     if "ERROR: file or directory not found: " in result.stderr:
+        print("test file or directory not found in this commit")
+        exit(0)
+    elif "ERROR: not found: " in result.stderr:
```

Comment:
this  condition happens when a new test method or function appear in an existing test file